src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -318,7 +335,11 @@ protected function instantiateObject(array &$data, $class, array &$context, \Ref
                 }
             }
 
-            return $reflectionClass->newInstanceArgs($params);
+            if ($constructor->isConstructor()) {
```

Comment:
This is breaking and should also be called if `$constructor` is null. However that throws an exception if the class has no constructor, so what about calling `newinstancewithoutconstructor` then?, being the absolute fallback.